File: docs/competitors/C++/sumNumbers/sumNumbers.cpp
//SumNumbers — читает последовательность действительных чисел и печатает сумму чисел

#include <iostream>
#include <sstream>
#include <string>
using namespace std;

int main() 
{
    double sum = 0.0;
    double num;
    int count = 0;
    string input;

    cout << "Enter numbers in one line separated by spaces:" << endl;

    getline(cin, input);

    stringstream ss(input);

    while (ss >> num) 
    {
        sum += num;
        count++;
    }

    if (count == 0) 
    {
        cout << "Numbers were not entered." << endl;
    }
    else 
    {
        cout << "Sum of " << count << " numbers: " << sum << endl;
    }

    return 0;
}

File: docs/competitors/Python/sumNumbers/sumNumbers.py
#SumNumbers — читает последовательность действительных чисел и печатает сумму чисел

input_str = input("Enter numbers in one line separated by spaces: ")
    
numbers = []
parts = input_str.split()
    
for part in parts:
    try:
        num = float(part)
        numbers.append(num)
    except ValueError:
        break
    
if numbers:
    total = sum(numbers)
    print(f"Sum of {len(numbers)} numbers: {total}")
else:

    print("No valid numbers were entered.")

